device-types/FS/FMU-MC084761EM3.yaml

```diff
@@ -0,0 +1,50 @@
+---
+manufacturer: FS
+model: FMU-MC084761EM3
+slug: fmu-mc084761em3
+comments: '[8ch Dual Fiber 1470-1610nm CWDM Mux Demux + Monitor, Expansion and 1310nm Port END OF SALE](https://www.fs.com/de/products/72428.html)'
+part_number: '72428'
```

Comment:
According to their product page, the FS P/N is `FMU-MC084761EM` so I think we should make the pn that. What do you think?